Directory.Packages.props

```diff
@@ -0,0 +1,75 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Azure.AI.FormRecognizer" Version="4.1.0" />
+    <PackageVersion Include="Azure.Identity" Version="1.10.0" />
+    <PackageVersion Include="Azure.Search.Documents" Version="11.5.0-beta.4" />
+    <PackageVersion Include="Azure.Storage.Blobs" Version="12.17.0" />
+    <PackageVersion Include="Azure.Storage.Queues" Version="12.15.0" />
+    <PackageVersion Include="DocumentFormat.OpenXml" Version="2.20.0" />
+    <PackageVersion Include="HtmlAgilityPack" Version="1.11.52" />
+    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.1.1" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.Binder" Version="7.0.4" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting" Version="7.0.1" />
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.1" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.TraceSource" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.SemanticKernel" Version="0.23.230906.2-preview" />
+    <PackageVersion Include="Microsoft.SemanticMemory.Core" Version="0.2.230901.2-preview" />
```

Comment:
I would avoid this row, it will be confusing having both a link to project and to the nuget

Reply:
Not sure I understand this. Package and project references use two different nodes (`PackageReference` vs `ProjectReference`). Since the Semantic Memory package is being referenced in at least once place I feel it makes sense to have at least one version defined centrally in case future projects need to use it. It can still be overridden on a per project basis or ignored completely (and the local project used) by using `ProjectReference`.

Reply:
We use the nuget only in one test project, as an example, and for now we don't have plans about referencing the nuget from within the repo (aside from the example). We might need examples to use different versions for other future reasons, so I would prefer leaving this out. I find it also confusing and it could lead to unexpected headaches debugging an unexpected dependency on the package rather than the code. Using this nuget is very much an exception, and it's better not having it here.

Reply:
Removed.